docs/en/observability/synthetic-monitoring.asciidoc

```diff
@@ -7,9 +7,9 @@
 
 [WARNING]
 ====
-This functionality is experimental and may be changed or removed completely in a future release.
-Elastic will take a best effort approach to fix any issues, but experimental features are not subject
-to the support SLA of official GA features.
+This functionality is in beta and is subject to change. The design and code is less mature
+than official GA features and is being provided as-is with no warranties. Beta features are
+not subject to the support SLA of official GA features.
```

Comment:
Any reason why we're maintaining all of these beta warnings instead of just using the `beta[]` tag like you added in `observability-introduction.asciidoc`? I'm guessing it's so that we can add the "Have a question? Want to leave feedback...". If you think that's worth keeping, I'd fix the beta[] label on in `observability-introduction.asciidoc` to include this

Reply:
@bmorelli25 Exactly. What's your thoughts on removing the "Have a question? Want to leave feedback..." content? Is it something we do in other docs?

Reply:
My recommendation would be to swap all of them to `beta[]` and add a TIP only on the first page that has the feedback link. We sometimes add links to landing pages, but never to all pages

Reply:
FWIW, I think it made sense during the alpha

Reply:
Great. Thanks!